mathesar_ui/src/api/users.ts

```diff
@@ -12,6 +12,7 @@ export interface UnsavedUser {
   email: string | null;
   username: string;
   password: string;
+  preferred_language: string;
```

Comment:
Can we type this as `Locales` instead of `string`? I haven't looked deeply, but it seems like that might eliminate the need for the type assertion `as Locales` that you have in `UserDetailsForm.svelte`, and it would maybe allow us to also use `Locales` instead of `string` in `SelectPreferredLanguage.svelte`

Reply:
75a5e4b